### ISSUE TITLE
[ADD FEATURE]: Implement Favorites Feature

### ISSUE THREAD

**Issue by Anushka-Pote** [Avdhesh-Varshney/Chanakya-Niti#94]
Develop a Favorites feature that allows users to save and access their preferred content, even without a sign-in system for user to access the liked content again or even with sign in functionality ([FEATURE] : User Sign-Up and Profiles
[#25](https://github.com/Avdhesh-Varshney/Chanakya-Niti/issues/25) is already assigned)

**Comment by Avdhesh-Varshney** [comment]
@Anushka-Pote are you working on it? and still you have not mention in which open source program are you participated?

**Comment by Anushka-Pote** [comment]
Yes I am already on it, and will be sending the PR this week itself. I am
participating as a contributor  under the Summer Social Code of Conduct
24 (SSOC).

**Comment by Anushka-Pote** [comment]
Hey @Avdhesh-Varshney 
For submission, do I even have to send some video or screenshot for the changes I made or is the PR request only enough?
